components/items/ItemsGrid/ItemsGrid.vue

```diff
@@ -62,6 +62,9 @@ import ItemsGridImage from './ItemsGridImage.vue'
 import { useFetchSearch } from './useItemsGrid'
 
 const isLoading = ref(true)
+const route = useRoute()
+route.query.listed = 'true'
```

Comment:
I'm afraid it's not an ideal solution. What if user visit the url with `?listed=felse`


`https://canary.kodadot.xyz/ksm/explore/items?listed=false`

Reply:
You are correct. I forgot to add an if condition `route.query.listed == undefined`

Anyway, let's discuss a better implementation on this with @vikiival